django_nasa/nasa/admin.py

```diff
@@ -0,0 +1,2 @@
+"""Admin panel function."""
```

Comment:
is there no admin panel in your project?
do you need an empty admin.py file though? is it compulsory to have it? (idk didnt try)